src/shared/components/map/UseReviewModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useCallback } from 'react';
+
+const useReviewModal = (apiEndpoint: string) => {
+    const [reviews, setReviews] = useState<string[] | undefined>(undefined);
+  const [storeName, setStoreName] = useState<string>("");
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
사소한 부분이지만 initial value가 존재하는 경우에 대한 타입 정의를 통일해주는 것이 좋습니다!
개인적으로는 initial value가 존재하는 경우 타입 정의를 굳이 하지 않는 쪽을 선호합니다!
```suggestion
  const [storeName, setStoreName] = useState("");
  const [isModalOpen, setIsModalOpen] = useState(false);
  // 또는
  const [storeName, setStoreName] = useState<string>("");
  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
```

Reply:
아하 감사합니다!